.github/workflows/pr_and_main.yml

```diff
@@ -79,14 +79,11 @@ jobs:
       - run: ctest --preset ci
       - run: python -m pytest -v tests
 
-      - name: Docs build
+      - run: cmake --build --preset docs
```

Comment:
How does the preset compare to `cmake --build . --target docs`? Do we need both?